portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,8 +32,6 @@
 /** Servlet that returns some example content. */
```

Comment:
This comment could be clearer; this is the servlet that displays all the comments from the datastore, correct?